tests/flow/test_load_csv.py

```
@@ -227,3 +252,42 @@ def test07_load_multiple_files(self):
+        result = self.graph.query(query, {'url': url}).result_set
+        for row in result:
+            self.env.assertIn(row[0], data)
+
+    def test_02_none_existing_url(self):
+        query = "LOAD CSV FROM $url AS row RETURN row"
+        urls = ["https://fakljsmndklnmsdvnkndqw02emkl.dodndasno12.dal/text.csv"]
+
+        for url in urls:
+            try:
+                self.graph.query(query, {'url': url})
+                self.env.assertFalse("we should have failed" and False)
```

Ruff (0.8.2)

Use `False` instead of `... and False`

Replace with `False`

(SIM223)